test/functional/test_framework/util.py

```diff
@@ -552,38 +552,31 @@ def gen_return_txouts():
 
 # Create a spend of each passed-in utxo, splicing in "txouts" to each raw
 # transaction to make it large.  See gen_return_txouts() above.
-def create_lots_of_big_transactions(node, txouts, utxos, num, fee):
-    addr = node.getnewaddress()
+def create_lots_of_big_transactions(mini_wallet, node, fee, tx_batch_size, txouts, utxos=None):
```

Comment:
nit: `create_lots_of_big_transactions` is not introduced in this PR, but maybe include "send" in the name, to clearly indicate functionality?